mne/label.py

```diff
@@ -747,6 +747,9 @@ def center_of_mass(self, subject=None, restrict_vertices=False,
         subject = _check_subject(self.subject, subject)
         if np.any(self.values < 0):
             raise ValueError('Cannot compute COM with negative values')
+        if np.all(self.values == 0):
+            raise RuntimeWarning('Label values are all 0, treating as a structural label\n')
```

Comment:
You want to use `warn` from `mne.utils` as `warn('Label values ...')`.

But really I think we can just make it an error, and suggest that the user (themselves) do `label.values[:] = 1.`. It makes the code a bit more explicit. (Plus the solution here modifies the user's object inplace, which isn't great.)